joint_trajectory_controller/include/joint_trajectory_controller/trajectory.hpp

```diff
@@ -149,7 +157,17 @@ class Trajectory
   rclcpp::Time time_before_traj_msg_;
   trajectory_msgs::msg::JointTrajectoryPoint state_before_traj_msg_;
 
-  bool sampled_already_ = false;
+  bool sampled_already_;
+
+  // For Ruckig jerk-limited smoothing
+  std::unique_ptr<ruckig::Ruckig<ruckig::DynamicDOFs>> smoother_;
+  ruckig::InputParameter<ruckig::DynamicDOFs> ruckig_input_{0};
```

Comment:
This was probably my own code but I think the `0` argument is hard to understand. Let's add a constant at the top of the file and use it here:

```
// Allow Ruckig objects to be initialized with dynamic size
constexpr size_t RUCKIG_DYNAMIC_DOFS = 0;
```

```suggestion
  ruckig::InputParameter<ruckig::DynamicDOFs> ruckig_input_{RUCKIG_DYNAMIC_DOFS};
```